pygotham/api/sponsors.py

```diff
@@ -0,0 +1,32 @@
+"""Sponsor-related API endpoints."""
+
+from flask import Blueprint
+from flask_restful import Api, Resource
+
+from pygotham.events.models import Event
```

Comment:
No particular reason. This was based on the schedule API module, which imports `Event` and `Talk` separately. I'm open to changing it; we should standardize on one approach.